cmd/kpod/create.go

```diff
@@ -183,6 +192,56 @@ func createCmd(c *cli.Context) error {
 	return nil
 }
 
+const seccompDefaultPath = "/etc/crio/seccomp.json"
+
+func parseSecurityOpt(config *createConfig, securityOpts []string) error {
+	var (
+		labelOpts []string
+		err       error
+	)
+
+	for _, opt := range securityOpts {
+		if opt == "no-new-privileges" {
+			config.noNewPrivileges = true
+		} else {
+			var con []string
+			if strings.Contains(opt, "=") {
+				con = strings.SplitN(opt, "=", 2)
+			} else if strings.Contains(opt, ":") {
+				con = strings.SplitN(opt, ":", 2)
+				logrus.Warn("Security options with `:` as a separator are deprecated and will be completely unsupported in 1.13, use `=` instead.")
```

Comment:
I think we can kill the Warn here. And maybe remove parsing of security-opt with colon separators entirely here?

Reply:
Ok

Reply:
Fixed